src/traces/isosurface/attributes.js

```diff
@@ -236,34 +235,20 @@ colorscaleAttrs('', {
 
     colorbar: colorbarAttrs,
 
-    // Flat shaded mode
-    flatshading: {
-        valType: 'boolean',
-        role: 'style',
+    opacityscale: {
+        valType: 'enumerated',
+        values: ['min', 'max', 'extremes', 'center'],
```

Comment:
Hmm. Could we also support 2D arrays similar to `colorscale` e.g.

```js
opacityscale: [[0, 0], [0.5, 0.2], [1, 0.9]]
```

?

Reply:
Yes I think we could/should. However; from the user perspective it might be easier to select a `function` that could be linear or non-linear compared to declaring `linear` opacity values that way.

Reply:
Right, the two signatures would be supported. Users could either use 

```js
opacityscale: 'Center';

// or
opacityscale: [[0, 0], [0.5, 0.2], [1, 0.9]];
```

similar to how `colorscale` has special string value e.g. `Reds`, `Viridis` ...

Reply:
Good pointer. 
Working on this now...